doc/expressionLanguage.md

```diff
@@ -17,6 +17,39 @@ The IoTAgent Library provides an expression language for measurement transformat
 information coming from the South Bound APIs to the information reported to the Context Broker. Expressions in this
 language can be configured for provisioned attributes as explained in the Device Provisioning API section in the
 main README.md.
+As an alternative, the IoTAgent Library supports as well [JEXL](https://github.com/TomFrost/jexl).
+To use JEXL, you will need to either configure it as default
+language using the `defaultExpressionLanguage` field to `jexl` or configuring
```

Comment:
```suggestion
language using the `defaultExpressionLanguage` field (see [configuration documentation](installationguide.md)) to `jexl` or configuring
```

to include a link to the place where `defaultExpressionLanguage` is explained in more detail.

Reply:
will fix

Reply:
fix by 3b6824b